src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static addToAlive(animal) {
+    Animal.alive.push(animal);
+  }
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
health should be 100 for every animal by default
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
```